spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -28,6 +29,13 @@ class DebuggingWidget(RichJupyterWidget):
     Spyder
     """
 
+    def __init__(self, *args, **kwargs):
+        super(DebuggingWidget, self).__init__(*args, **kwargs)
+        # Adapted from qtconsole/frontend_widget.py
```

Comment:
Please clarify what the line below this one does in this comment. Is it only highlighting `ipdb>`? or is it doing something else?

Reply:
It marks `ipdb> ` as a prompt